src/convertToObject.js

```diff
@@ -3,16 +3,35 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesAndProp = sourceString
+    .split(';')
+    .map(item => item.split(': '));
+
+  const reducer = stylesAndProp
+    .filter(item => item.length > 1)
+    .reduce((acc, item) => {
+      const key = item[0].replace('\n', '').trim();
+      const values = item
+        .splice(1).map((value) => value.replace('\n', '').trim());
```

Comment:
why don't you just do the same thins as you did with the key?
```suggestion
      const key = item[0].replace('\n', '').trim();
      const value = item[1].replace('\n', '').trim();
```